scripts/reborn_webui_v2_live_qa/run_live_qa.py

```
@@ -4059,6 +4182,117 @@ async def run_gated(ctx: LiveQaContext) -> ProbeResult:
+        marker=marker,
+        required_text=[],
+        timeout=240.0,
+    )
+    if not result.success:
+        return result
+    reply_text = str(result.details.get("text_excerpt") or "")
+    leaked_ids = _raw_slack_user_ids_in_text(reply_text)
+    if leaked_ids:
+        return _result(
+            case_name,
+            False,
```

Ruff (0.15.20)

[warning] Boolean positional value in function call

(FBT003)